README.md

```diff
@@ -1 +1,1205 @@
+[main]     found 0 vulnerabilities
+[main]     npm
+[main]      WARN using --force
+[main]      I sure hope you know what you are doing.
+[main]
+[main]      ---> 9ff0e005febd
+[main]     Removing intermediate container e0a52c6e7d2e
+[main]     Step 5/7 : COPY . ./
+[main]      ---> 211ef0e15576
+[main]     Removing intermediate container 75e2127552bf
+[main]     Step 6/7 : RUN cat /run/secrets/my-secrets
+[main]      ---> Running in 949098cba8b6
+[main]     This file has build-time secrets!
+[main]      ---> 6da25ec8f94f
+[main]     Removing intermediate container 949098cba8b6
+[main]     Step 7/7 : CMD npm start
+[main]      ---> Running in 1ae564a36e44
+[main]      ---> e590faac0013
+[main]     Removing intermediate container 1ae564a36e44
+[main]     Successfully built e590faac0013
+[Info]     Uploading images
+[Success]  Successfully uploaded images
+[Info]     Built on arm03
+[Success]  Release successfully created!
+[Info]     Release: cedeff39dc537c9cd4c1df3b97ffcbc7 (id: 1055448)
+[Info]     ┌─────────┬────────────┬────────────┐
+[Info]     │ Service │ Image Size │ Build Time │
+[Info]     ├─────────┼────────────┼────────────┤
+[Info]     │ main    │ 208.47 MB  │ 11 seconds │
+[Info]     └─────────┴────────────┴────────────┘
+[Info]     Build finished in 46 seconds
+			    \
+			     \
+			      \\
+			       \\
+			        >\/7
+			    _.-(6'  \
+			   (=___._/` \
+			        )  \ |
+			       /   / |
+			      /    > /
+			     j    < _\
+			 _.-' :      ``.
+			 \ r=._\        `.
+			<`\\_  \         .`-.
+			 \ r-7  `-. ._  ' .  `\
+			  \`,      `-.`7  7)   )
+			   \/         \|  \'  / `-._
+			              ||    .'
+			               \\  (
+			                >\  >
+			            ,.-' >.'
+			           <.'_.''
+			             <'
+```
+As you can see, step 6 output:
+```
+[main]     Step 6/7 : RUN cat /run/secrets/my-secrets
+[main]      ---> Running in 949098cba8b6
+[main]     This file has build-time secrets!
+```
+which is the contents of the secrets file. This file could obviously contain
+a raft of different functions, including a script that gets executed, text
+for filling in files, etc. As well as defining globally accessible secrets
+(which are shared to all services being built), there is also the option to
+define secrets that are only accessible to particular services, or to map them
+to different paths. This becomes useful in multicontainer build scenarios. This
+can be achieved by adding a `services` section to the `balena.yml` file. For
+example:
+```
+services:
+  main:
+    - source: main-only-secrets
+	  dest: my-main-secrets
+```
+This change would map the `.balena/secrets/main-only-secrets` file into the
+`/run/secrets/my-main-secrets` runtime path at build-time but *only for the
+`main` service*.
+
+Another frequent use for secrets at build-time is that of environment variables
+that are defined but shouldn't exist in the generated service image. Much in
```

Comment:
This is wrong, and could be pretty dangerous. Build arguments can be retrieved from a built image, so we strongly recommend not using them for anything secret

Reply:
I think I'd misread the documentation 'Build Time Secrets and Variables' as a heading to me implies both secrets and variables are build time only, but it is disputed further down. I think this heading needs changing in the docs.

Reply:
Done.